frontend/src/lib/components/Users/UserModal.svelte

```diff
@@ -119,6 +121,21 @@
         <h3>{$t('admin_dashboard.user_details_modal.createdBy')}</h3>
         <p class="value">{user.createdBy?.name  ?? NULL_LABEL}</p>
       </div>
+      <AdminContent>
+      <div>
+        Feature flags:
+        <ul>
+          {#each allPossibleFlags as flag}
+            <li><input
+              type="checkbox"
+              name="featureFlags"
+              value={flag}
+              bind:group={user.featureFlags}
+            > {flag}</li>
```

Comment:
I think these might need to be disabled or something, right now I can click on them to change them which we probably don't want in this view

Reply:
Yeah, I think I'm going to change how they're presented in the user modal to be just a comma-separated list of strings rather than checkboxes.

Reply:
Done in commit 1a73ef4b.